libs/ngx-mime/src/lib/core/intl/help-intl.no_nb.ts

```diff
@@ -12,5 +12,6 @@ export class HelpIntlNoNb extends HelpIntl {
   line8 = '<strong>P</strong>: Går til forrige treff i objektet';
   line9 = '<strong>F</strong>: Fullskjerm av og på (Lukk med <strong>F</strong> eller <strong>ESC</strong>-tasten.)';
   line10 = '<strong>R</strong>: Rotér 90°';
+  line11 = '<strong>T</strong>:  Vis/skjul optisk gjenkjent tekst (for innhold som kan vise det)';
```

Comment:
To get screen readers to pause after an sentence, we should have a period punctuation at the end of each sentence.
Parenthesis are not read by the screen readers. 
PS: I have only tested with a free [Chrome extension](https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn)

```suggestion
  line11 = '<strong>T</strong>:  Vis/skjul optisk gjenkjent tekst. (for innhold som kan vise det).';
```